argocd/efs-provisioner/Chart.yaml

```diff
@@ -2,6 +2,10 @@
 apiVersion: "v2"
 name: "efs-provisioner"
 version: "0"
+description: |
+  The chart installs provisioner enabling:
+
+  * mounting EFS storage as PersistentVolumes in kubernetes
```

Comment:
Actually, it provides a StorageClass for EFS, along with a provisioner, so you can get dynamic PVC on EFS without manually creating PVs.

Reply:
Also note it's specific to AWS, of course

Reply:
updated text followin suggestions